data/meterpreter/meterpreter.php

```diff
@@ -419,8 +425,44 @@ function core_loadlib($req, &$pkt) {
 }
 
 
+function core_uuid($req, &$pkt) {
+    my_print("doing core_uuid");
+    packet_add_tlv($pkt, create_tlv(TLV_TYPE_UUID, PAYLOAD_UUID));
+    return ERROR_SUCCESS;
+}
 
 
+function get_hdd_label() {
+  foreach (scandir('/dev/disk/by-id/') as $file) {
```

Comment:
The order these value come out is different to Python and POSIX, which means that the Machine ID is different for PHP payloads. Should we add some kind of sorting to this, or look for another thing to do to make it consistent?